src/modules/auth/containers/LoginCard.js

```diff
@@ -72,7 +72,7 @@ function LoginCard(props) {
 
 	// Check on insecured connection
 	if (!insecuredConnection && location.protocol === 'http:' && location.hostname !== 'localhost' && location.hostname !== '127.0.0.1') {
-		props.app.addAlert('warning', t('LoginCard|Insecured connection'));
+		props.app.addAlert("warning", `${t("LoginCard|Insecured connection")}`, 30);
```

Comment:
@Aringoaway Why did you wrap it into dynamic string representation? It has no use here since we dont pass any dynamic value to the alert message